.travis.yml

```diff
@@ -3,6 +3,5 @@ sudo: false
 node_js:
   - 4
   - 6
-  - 7
```

Comment:
Why remove node 7?

Reply:
Node 7 is end of life https://github.com/nodejs/Release#release-schedule